src/components/Devices/Devices.react.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import susi from '../../images/susi.svg';
+import susiAndroid from '../../images/susi-test.gif';
+import susiiOSGif from '../../images/ios_demo.gif';
+import susiDesktop from '../../images/susi.gif';
+import './Devices.css';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import StaticAppBar from '../StaticAppBar/StaticAppBar.react';
+
+class Devices extends Component{
+  constructor(props){
+    super(props);
+    this.state={
+      video:false,
+    }
+  }
+
+  render() {
+
+    return (
+      <div>
+
+        <StaticAppBar {...this.props}
+          location={this.props.location} closeVideo={this.closeVideo} />
+
+        <div className='head_section'>
+          <div className='container'>
+            <div className="heading">
+              <h1>Devices</h1>
+              <p>SUSI.AI is available on every platform</p>
+            </div>
+          </div>
+        </div>
+        <div className="device_section">
+          <div className="device_description">
+            <div className="heading">Android devices</div>
+            <p className="text">
+              SUSI.AI is available on android devices.
+              Download the Android App to have access to SUSI on the go.
+            </p>
+            <div className="store_icons">
+              <a href="https://github.com/fossasia/susi_android/raw/apk/susi-debug.apk"
+                className="play_store">Get It on Google Play</a>
+            </div>
+          </div>
+          <div className='img-container'>
+            <img src={susiAndroid} alt='susi-android' className='susi-test' />
+          </div>
+        </div>
+        <div className="device_section">
+          <div className="device_description">
+            <div className="heading">iOS devices</div>
+            <p className="text">
+              SUSI.AI is available on iOS devices.
+              Download the iOS App to have access to SUSI on the go.
+            </p>
+            <div className="store_icons">
+              <a href="https://github.com/fossasia/susi_iOS/"
+                className="app_store">Download on the App Store</a>
+            </div>
+          </div>
+          <div className='img-container'>
+            <img src={susiiOSGif} alt='susi-iOS' className='susi-test' />
+          </div>
+        </div>
+        <div className="device_section bottom_section">
+          <div className="device_description">
+            <div className="heading">Desktops and Laptops</div>
+            <p className="text">
+              You can use SUSI.AI Web Chat on your desktop and laptop.
+              Start Chatting with SUSI.AI
+              <Link style={{ textDecoration: 'none'}}
```

Comment:
Hey, since the person is already viewing this page through chat.susi.ai maybe we should just redirect him to the landing page instead of opening a new tab? @rishiraj824 @isuruAb Please provide your thoughts on this